recipe-server/normandy/settings.py

```diff
@@ -162,7 +164,15 @@ def CSP_DEFAULT_SRC(self):
     CSRF_COOKIE_NAME = 'csrftoken-20170707'
 
 
-class Base(Core):
+class CORS:
+    """Default settings related to setting CORS headers."""
+
+    CORS_ORIGIN_ALLOW_ALL = values.BooleanValue(True)
+
```

Comment:
As a very minor nit, I don't think this blank line is needed. Feel free to leave it the way it is though.

Reply:
sure thing. consistency.